### ISSUE TITLE
Associate the Arrows with Navigational Groups

### ISSUE THREAD

**Issue by pablohoneyhoney** [WordPress/wporg-main-2022#128]
The arrow should be closer to the navigational instance it’s associated to. Currently they are at an uncanny distance that creates a scattered visual effect, but also doesn't make the taxonomies clear. 

<img width="709" alt="Screen Shot 2022-09-19 at 11 30 20 AM" src="https://user-images.githubusercontent.com/25516611/190989083-7e6a3b23-fbbb-435c-ad34-1bd8cd7bfa64.png">
<img width="690" alt="Screen Shot 2022-09-19 at 11 30 12 AM" src="https://user-images.githubusercontent.com/25516611/190989084-8a1c4043-544c-4a30-a8a0-8b78def5afb9.png">

**Comment by jasmussen** [comment]
I would agree! It looks separate at this point. Something like this, maybe?

<img width="1007" alt="Screenshot 2022-09-19 at 11 36 02" src="https://user-images.githubusercontent.com/1204802/190990093-3cc75273-8ce7-498c-ad7a-786e3be6b5a9.png">

**Comment by ryelle** [comment]
We talked about this a little in https://github.com/WordPress/wporg-mu-plugins/pull/263 & https://github.com/WordPress/wporg-mu-plugins/issues/261 — the arrows need to have enough space to be touch targets (illustrated here with the focus state).

<img width="619" alt="Screen Shot 2022-09-19 at 10 44 42 AM" src="https://user-images.githubusercontent.com/541093/191045231-a9261031-2d15-485a-950f-2d365510c0e9.png">

The right side of the text already has a less spacing than the left, so I don't think we can shift that any more.

<img width="365" alt="Screen Shot 2022-09-19 at 10 45 29 AM" src="https://user-images.githubusercontent.com/541093/191045466-1336a843-05b4-474c-a7d6-140524c0c5e6.png">

@javierarce walked through a few ideas [in this comment](https://github.com/WordPress/wporg-mu-plugins/pull/263#issuecomment-1237180956), but ultimately we landed on what you see now.

**Comment by ryelle** [comment]
Maybe there could be more_space between the menu items to make it clear the text + chevron are a unit?
